openlibrary/templates/lists/widget.html

```diff
@@ -7,7 +7,7 @@
 $ username = ctx.user and ctx.user.key.split('/')[-1]
 $ users_work_read_status = read_status or work.get_users_read_status(username) if work and username else None
 $ users_work_rating = rating or work.get_users_rating(username) if work and username else None
-
+$ pidentifier = page.url() if page.url().startswith("/publishers/") else None
```

Comment:
Do we want to blacklist publishers or whitelist the supported types (works, editions, authors, & subjects?). I think subjects are currently allowed, but I'm not sure they should be.

Reply:
This shouldn't be necessary.
The template openlibrary/templates/publishers/view.html can avoid calling the macro when the list is not wanted.